packages/shared/src/dto/subscriber/get-subscriber-preferences.dto.ts

```diff
@@ -0,0 +1,20 @@
+import {
+  IPreferenceChannels,
+  IPreferenceOverride,
+} from '../../entities/subscriber-preference/subscriber-preference.interface';
+
+export interface IGetSubscriberPreferencesResponseDto {
+  global: {
+    enabled: boolean;
```

Comment:
Fun fact:
we allow `enabled` in update DTO, to disable all global preferences as I understand, but we never use it anywhere:

```tsx
export type UpdateSubscriberGlobalPreferencesRequestDto = {
  /**
   * Enable or disable the subscriber global preferences.
   */
  enabled?: boolean | undefined;
  /**
   * The subscriber global preferences for every ChannelTypeEnum.
   */
  preferences?: Array<ChannelPreference> | undefined;
};
```

we always return `enabled: true` so you can't in fact disable all preferences globally.
https://github.com/novuhq/novu/blob/368050c27066041996769ea83b1d7900f1ffce83/libs/application-generic/src/usecases/get-subscriber-template-preference/get-subscriber-template-preference.usecase.ts#L110